### PR TITLE
added setTime.md

### LINKED ISSUE #2501: [Term Entry] JavaScript Dates .setTime()
### Reason/inspiration (optional)

We would like a new term entry for JavaScript [Dates](https://www.codecademy.com/resources/docs/javascript/dates) about the `.setTime()` method. The entry should go in a new file in `docs/content/javascript/concepts/dates/terms/setTime/setTime.md`

The entry should
    - Have an introductory paragraph about what the method is and does.
    - An example showing the method in use.

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by ShivangRawat30**
### Description

Added setTime at `docs/content/javascript/concepts/dates/terms/setTime/setTime.md`
Closes #2501

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/dates/terms/setTIme/setTIme.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  - 'Web Development'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'paths/web-development'
+  - 'introduction-to-javascript'
+---
+
+The **`.setTime()`** method returns a number, representing the number of milliseconds between the date object and midnight January 1 1970.
```
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setTIme/setTIme.md:14]:
```suggestion
The **`.setTime()`** method sets date and time by adding or subtracting a specified number of milliseconds to/from midnight January 1, 1970.
```
The intro sentence and description should be in sync.

File: content/javascript/concepts/dates/terms/setTIme/setTIme.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+In the example below, the `myDate` variable takes the provided Time and then the `.setTime()` method is applied to return a modified time.
+
+```js
+const myDate = new Date();
+myDate.setTime(1332403882588);
+
+console.log(myDate);
+```
+
+This results in the following output:
```
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setTIme/setTIme.md:35]: Note this output will be localized, set in the user's time zone.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@ShivangRawat30 I did a fist review and had a couple of comments.

**Review by SSwiniarski — APPROVED** [review]
@ShivangRawat30 Looks ready for a second review.

**Comment by SSwiniarski** [comment]
@ShivangRawat30, we've gone through a review so we're going to keep this PR, but you are **not** supposed to submit a PR for an issue you have not been assigned. There was someone already waiting to contribute to this issue. Do **not** do this again, we have assignments to prevent this situation, where two people are working on the same issue.

**Review by caupolicandiaz — APPROVED** [review]
@ShivangRawat30 @SSwiniarski This is all set to merge. I just made a couple of minor edits, including renaming the file and folder which were set as setTIme in lieu of setTime.

**Comment by SSwiniarski** [comment]
@ShivangRawat30 Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/javascript/dates/setTime

image

You're currently anonymous. If you don't wish to be anonymous, check out this forum thread on how to link your GitHub and Codecademy profiles: https://discuss.codecademy.com/t/my-contribution-is-live-but-im-not-being-shown-as-a-contributor-why-is-this/758036  **Note:** it may take a few minutes for the change to register on the entry page.

**Pull request merged**